### ISSUE TITLE
nextvlad

### ISSUE THREAD

**Issue by zhangatao** [ceshine/yt8m-2019#4]
Hello, thank you for opening up your code. Now there is a problem. I am using your nextvlad code to classify video at the frame level. The effect is not as good as simple pooling. What causes it?

**Comment by zhangatao** [comment]
My data and loss function are consistent with https://github.com/linrongc/youtube-8m/blob/master/nextvlad.py, using simple pooling, the gap can reach 0.8, but after using nextvlad , gap can only reach 0.2

**Comment by ceshine** [comment]
As [stated in the docstring](https://github.com/ceshine/yt8m-2019/blob/master/yt8m/encoders.py#L144), my NeXtVLAD implementation is adapted from `linrongc/youtube-8m` as you mention. However, I don't really remember what changes did I make can break the compatibility with the original pipeline. I'd suggest you make sure that your pipeline is compatible with [my implementation](https://github.com/ceshine/yt8m-2019/blob/master/yt8m/dataloader.py).

**Comment by zhangatao** [comment]
Thanks🙏
